packages/lockfile-file/src/sortLockfileKeys.ts

```diff
@@ -38,6 +39,7 @@ const ROOT_KEYS_ORDER = {
   overrides: 3,
   packageExtensionsChecksum: 4,
   patchedDependencies: 5,
+  [InlineSpecifiersFormatLockfileKey]: 6,
```

Comment:
we have a lockfile version field already. Maybe just use a new version for it. Like `v6.0-alpha.0`

Reply:
I can do that. Was a bit worried about that making it harder to back out this change if we don't want to introduce this in v6.0?

What are your thoughts on `vCURRENT-inlineSpecifiers`? That makes it harder for multiple lockfile format feature flags. I think that's very unlikely, but wanted to keep that possible. Happy to just discard the extra field and reuse the version field.

Reply:
`5.4-inlineSpecifiers` sounds good to me as well.

Reply:
Should be done in latest push.